controllers/topology/internal/extensions/patches/inline/json_patch_generator_test.go

```diff
@@ -677,6 +677,153 @@ func TestCalculateValue(t *testing.T) {
 			},
 			want: &apiextensionsv1.JSON{Raw: []byte(`"value"`)},
 		},
+		// Objects
```

Comment:
If I'm not wrong we are missing error cases for objects and array

Reply:
Yup absolutely forget them, thx! I added a lot of tests here. I tried to add some for the render func, but it's hard to produce errors with go templating, it just renders a fallback value.